src/components/msfvenom/msfvenom.tsx

```diff
@@ -132,7 +132,7 @@ const payloadRequiredVariables = [
     ],
 ];
 
-const PayloadGenerator = () => {
+const payloadGenerator = () => {
```

Comment:
Please see the issue from the CI testing.

Error: src/components/msfvenom/msfvenom.tsx(455,16): error TS2552: Cannot find name 'PayloadGenerator'. Did you mean 'payloadGenerator'?

- [ ] Rectify error.

Thank you!